content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -61,12 +62,12 @@ To connect a MetaMask wallet to the bridge, go to the [EVM Sidechain bridge](htt
 
 ![Connect MetaMask Wallet](img/evm-sidechain-connect-metamask.png "Connect MetaMask wallet")
 
-### 2. Initiate the Transfer of Funds 
+### 2. Initiate the Transaction 
 
 Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount to send, and the destination address.
```

Comment:
Change from:
Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount to send, and the destination address.
To:
Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount and the address to fund.